projects/dynamicforms/src/core/connectors/file.adapter.ts

```diff
@@ -19,9 +19,9 @@ export abstract class FileAdapter {
    * @param userId The user id
    * @param fileCodes The file codes
    */
-  abstract getFilesForCodes(
+  abstract getFilesForUser(
```

Comment:
If you think that this is ok (to have fileCodes as optional param), i'll need to deprecate getFilesForCodes method and introduce this one

Reply:
Fixed